skills/mine-challenge/challenge-gate.md

```
@@ -0,0 +1,70 @@
+
+- **`<header>`** — the `AskUserQuestion` header chip, e.g. `Challenge`. Keep it ≤12 chars.
+- **`<gate_type>`** — the cfl gate type: `define-challenge`, `sketch-challenge`, or `ship-challenge`.
+- **`<target>`** — what to pass to `/mine-challenge`: a design doc path or a changed-files list file.
+- **`<critic_flag>`** — `--critics=N` if pinned, or empty string if using triage default.
+- **`<re_challenge_flag>`** — `--re-challenge` if this is a re-challenge, or empty string.
+- **`<post_resolution>`** — caller-specific handling after step 6 completes (e.g., sketch's upgrade-to-caliper check, ship-time's unresolved-finding summary).
+
+## The sequence
+
+1. Record the dispatch (skip if cfl tracking is inactive for this run):
+   ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)